packages/mobile/ios/Podfile.lock

```diff
@@ -247,6 +258,10 @@ DEPENDENCIES:
   - React-RCTVibration (from `../../../node_modules/react-native/Libraries/Vibration`)
   - ReactCommon/jscallinvoker (from `../../../node_modules/react-native/ReactCommon`)
   - ReactCommon/turbomodule/core (from `../../../node_modules/react-native/ReactCommon`)
+  - "RNCMaskedView (from `../../../node_modules/@react-native-community/masked-view`)"
```

Comment:
Why does it wrapped with ""?

Reply:
This text is automatically generated when we install packages. So this is not my records, this records are generated.